File: EDUARDO/decide_detector.m
classdef decide_detector < handle
   properties
    
   DbDecidido
       
   end
   methods
   % Construtor inializador do objeto
   % inival: inicia todas as variaveis
        function obj =decide_detector

            obj.DbDecidido = 0;
            
            
        end
    
	    function DbDecidido = decidido(obj,Db,Db2)
		
			if ((Db == 1) || (Db2==1))
        
				obj.DbDecidido=1;

			else

				obj.DbDecidido=0;
		
			end						  
         
         DbDecidido = obj.DbDecidido;
                
        end                   
        
   end
end

File: EDUARDO/APPs/AppCompleto_v22_NovoDetec/app/src/main/java/fun/eduardo/funcoes/decide_detector.java
package fun.eduardo.funcoes;

public class decide_detector {

	double DbDecidido;


	public decide_detector() {

		this.DbDecidido = 0;

	}


	public double decidido(double Db, double Db2) {

		if ((Db == 1) || (Db2 == 1)) {
			this.DbDecidido = 1;

		} else {

			this.DbDecidido = 0;

		}


		return DbDecidido;

	}

}
